docs/best_practices/todo.rst

```diff
@@ -0,0 +1,116 @@
+:orphan:
+
+This is an isolated page of the documentation containing all current section references autogenerated by the check to best practice linkage of the conf.py - this page is temporary and is not to be included in any index
+
+When all of these are done, this page will be removed.
```

Comment:
Explanatory description for this file. Otherwise the missing references rightly throw a warnings which is now an error.

Going through these now, but some actually do represent BP sections that are currently missing and are better to add in subsequent PRs.